src/ceph-volume/ceph_volume/util/system.py

```diff
@@ -297,7 +297,13 @@ def set_context(path, recursive=False):
         )
         return
 
-    stdout, stderr, code = process.call(['selinuxenabled'], verbose_on_failure=False)
+    try:
+        stdout, stderr, code = process.call(['selinuxenabled'],
+                                            verbose_on_failure=False)
+    except FileNotFoundError:
```

Comment:
$ head -1 /usr/sbin/ceph-volume
#!/usr/bin/python2.7

FileNotFoundError is not defined in python2 and the actual exception is 'OSError'. In my testing on Xenial I was able to get it to work by adding the following block before your 'try' block.

```
    try:
        FileNotFoundError
    except NameError:
        FileNotFoundError = OSError
```

Reply:
This issue is actually described in #31809 but oddly nothing was done about it.

Reply:
@badone There is a separate issue for this https://tracker.ceph.com/issues/42967. Here is the backport of it #32867

Reply:
@jan--f ack, thanks, that explains it.